Documentation/devel/user-namespaces.md

```diff
@@ -0,0 +1,48 @@
+User namespaces
+===============
+
+Background
+----------
+
+User namespaces is a feature of Linux that can be used to separate the user IDs
+and group IDs between the host and containers. It can provide a better
+isolation and security: the privileged user `root` in the container can be
+mapped to a non-privileged user on the host.
+
+Implementation status
+---------------------
+
+rkt has an initial experimental implementation based on systemd-nspawn. A pod
+can transparently use user IDs in the range 0-65535 and this range is mapped on
```

Comment:
@iaguis : since you are looking at that code at the moment, can you confirm the range is correct?